src/gtest/test_merkletree.cpp

```diff
@@ -293,7 +319,7 @@ TEST(merkletree, testZeroElements) {
     for (int start = 0; start < 20; start++) {
```

Comment:
Hang on… is it coincidence that this is 20? Now that I recheck it appears it's unrelated to the tree depth (as long as the tree can fit at least 20 leaves, so depth 5 or greater)?